### DISCUSSION TITLE
foundryup downloading up extremely slow

### DISCUSSION THREAD

**Discussion by sunjoke399** [Cyfrin/foundry-full-course-cu#2837] [General]
Everytime I run "foundryup" command it takes extremely long time (10~20 minutes) for "foundryup: downloading latest forge, cast, anvil, and chisel" progress bar to finish. I tried a macbook and another mac, both the same.  I am wondering if it has anything to do with my home network?  

When I install foundry with "curl -L https://foundry.paradigm.xyz | bash"  I encounter an error "curl: (7) Failed to connect to raw.githubusercontent.com port 443 after 16 ms: Couldn't connect to server". I fixed by adding in "185.199.108.133 raw.githubusercontent.com" in the /etc/hosts/ file.

**Comment by EngrPips** [comment]
Hello @sunjoke399, I am convinced it has something to do with your Home network as I ran `foundryup` now and it didnt take a minute to complete the whole process.

**Reply by sunjoke399** [reply]
thanks for the reply. yeah I physically change to another network and it become faster immediately, though still took about one minute.

**Reply by EngrPips** [reply]
One minute isn't bad hahaha
